Tests/MacroTestingTests/MacroExamples/EntryMacro.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftSyntax
+import SwiftSyntaxMacros
+
+// Not complete, just enough to unit test lexical context.
+public struct EntryMacro: AccessorMacro {
+  public static func expansion(
+    of node: AttributeSyntax,
+    providingAccessorsOf declaration: some DeclSyntaxProtocol,
+    in context: some MacroExpansionContext
+  ) throws -> [AccessorDeclSyntax] {
+    let isInEnvironmentValues = context.lexicalContext.contains { lexicalContext in
+      if let ext = lexicalContext.as(ExtensionDeclSyntax.self), ext.extendedType.trimmedDescription == "EnvironmentValues" {
+        return true
+      } else {
+        return false
+      }
```

Comment:
Editing on my phone so it's hard to tell if the white space is right, but:

```suggestion
      lexicalContext.as(ExtensionDeclSyntax.self)?.extendedType.trimmedDescription == "EnvironmentValues"
```